apps/src/calc/calc.js

```diff
@@ -199,26 +201,31 @@ Calc.init = function (config) {
     }
   };
 
-  var codeWorkspace = (
-    <CodeWorkspace
-      localeDirection={studioApp.localeDirection()}
-      editCode={!!level.editCode}
-      readonlyWorkspace={!!config.readonlyWorkspace}
-    />
-  );
+  // Push initial level properties into the Redux store
+  studioApp.reduxStore.dispatch(setInitialLevelProps({
+    localeDirection: studioApp.localeDirection(),
```

Comment:
That's a good point. This has kind of become initialProperties instead of initalLevelProperties. Do you think it makes sense to split these out as two modules, or just rename to make it clearer it's not only level properties?